api/src/helpers/trpc.ts

```diff
@@ -0,0 +1,23 @@
+import * as trpcExpress from '@trpc/server/adapters/express';
+import { TRPCError, initTRPC } from '@trpc/server';
+
+export const createContext = ({ req, res }: trpcExpress.CreateExpressContextOptions) => ({
```

Comment:
something with this type here is throwing a typescript error

Reply:
I'm not sure why it says `Property 'session' does not exist on type 'Request<ParamsDictionary, any, any, ParsedQs, Record<string, any>>` but the property definitely does exist

Reply:
(req as Request).session

Reply:
Express.Request.session: Session & Partial<SessionData>

Reply:
```ts
import { Request } from 'express';
import { Session, SessionData } from 'express-session';
import '../types/types';
```
```ts
session: (req as Request & { session: Session & Partial<SessionData> }).session,
```

Reply:
resolved: some issue with module augmentation for express-session